### DISCUSSION TITLE
Youtube Auth Token

### DISCUSSION THREAD

**Discussion by sonnydelight420** [jagrosh/MusicBot#1721] [Questions] [answered]
Hey everyone, I was wondering if it's possible to use the Youtube auth token feature to get around the issue with the current version of the bot. I am hosting the bot through BisectHosting.

PS. I'm very entry level on how java and the bot works. So I apologize if this question seems silly.

**Comment by MichailiK** [comment, marked as answer]
Signing into YouTube has not been implemented yet. It will be available in the next version of JMusicBot, 0.4.4.

**Reply by s0kalau** [reply]
Will there be any?
